.circleci/config.yml

```diff
@@ -123,21 +123,21 @@ workflows:
           context:
             - slack
     triggers:
-      - schedule: 
+      - schedule:
           cron: "0 * * * *"
           filters:
             branches:
               only:
                 - main
-  main:
+  pr-checks:
+    unless:
+        equal: [ main, << pipeline.git.branch >> ]
```

Comment:
I think we should run this on main too, that way if the branch wasn't up to date with main before merge we check right away to make sure changes are still valid, then it will run on the hourly job afterwards?

Reply:
Good shout. Another option for combatting this is to require PRs are up to date before merging. But this is good, too [run prs-check on main, too](https://github.com/ethereum-optimism/superchain-registry/pull/108/commits/dd78be34848a898e4c83acf9b8f8b5fc6066db09)

Reply:
I think we might have already have the up to date requirement? Let's see
<img width="882" alt="image" src="https://github.com/ethereum-optimism/superchain-registry/assets/17163988/8f00f510-b583-46c8-a31f-76c8dca983f7">

> We won't get the slack notification if this happens, BTW. Not until the next scheduled "hourly" job runs.

Good point. I think that is ok for now, especially if the above is correct. This way if the next hourly run does fail, and another PR happened to be merged in between, we'll have the run history on main to find the exact commit that caused the issue